packages/build-utils/src/types.ts

```diff
@@ -8,16 +9,13 @@ export interface Env {
   [name: string]: string | undefined;
 }
 
-export interface File {
+export type File = FileRef | FileFsRef | FileBlob;
```

Comment:
Should this include `Lambda`? I think that was the purpose of the `toStream()` and `toStreamAsync()` props here

Reply:
There are parts of the code that treat `Lambda` and `File*` the same, but they don't have properties in common. We think this should be separate. You can see some of the impact of this here: https://github.com/vercel/vercel/pull/7530#discussion_r821997250